src/test/groovy/com/athaydes/spockframework/report/internal/HtmlReportCreatorSpec.groovy

```diff
@@ -273,4 +298,18 @@ class HtmlReportCreatorSpec extends ReportSpec {
         }
     }
 
+    @Category( SpockReportExtension )
+    class ShowCodeBlocksEnabled {
+        static final htmlReportCreator = new HtmlReportCreator()
+        SpecInfoListener createListener() {
+            setShowCodeBlocks( true )
+            configReportCreator htmlReportCreator
+            new SpecInfoListener( htmlReportCreator )
+        }
+    }
+
+    @Category( SpockReportExtension )
+    class ShowCodeBlocksDisabled {
```

Comment:
I see you got how `Category` works :) Not sure if you're aware, but this has been deprecated, Groovy now has `trait`s which replace this... but don't change it, otherwise we'll need to rewrite all other categories! Just wanted to be sure you knew that so you don't use deprecated stuff in your own projects later because of me :D